packages/mdc-drawer/_mixins.scss

```diff
@@ -73,32 +75,26 @@
 
 @mixin mdc-drawer-icon-activated-ink-color($color, $opacity) {
   $value: rgba(mdc-theme-prop-value($color), $opacity);
-  // stylelint-disable selector-no-qualifying-type
-  .mdc-list-item--activated,
-  a.mdc-list-item--activated, {
+
+  .mdc-list-item--activated {
```

Comment:
I noticed you took out `a` selectors here, but there's still one in `mdc-drawer-standard.scss`; is it still needed?